changelog.d/300.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug that would cause membership queue failures to not be reported as metrics
```

Comment:
I don't understand what this means. Why are failures reported "as metrics"? Do you mean "as part of the metrics"?
I also don't understand how adding the label "type" fixes some failures not being reported.

Reply:
Update about 2): After having read the issue I found the code where "type" is being used as a label later.
I now understand that this is about handing over all labels on the initial sync of metrics.

Reply:
Sorta. Prom-client will get angry if you use a label not defined in the schema. This PR just ensures we define it. Have updated the changelog entry to make it a bit easier to see the bug?